src/devices/vdml_device.cpp

```diff
@@ -0,0 +1,44 @@
+/**
+ * \file devices/vdml_device.cpp
+ *
+ * Base class for all smart devices.
+ *
```

Comment:
lol be sure to add errno vals here

Reply:
(Oh crap nevermind I am stupid... this was not necessary. I'm insanely sorry don't let me do late night code review ever again )